services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeGeometriaControllerITests.kt

```diff
@@ -1,12 +1,14 @@
 package fi.hel.haitaton.hanke
 
 import com.ninjasquad.springmockk.MockkBean
+import fi.hel.haitaton.hanke.organisaatio.OrganisaatioService
```

Comment:
Both the added imports are now unnecessary.

Reply:
Thanks, removed them!